src/dotnet/commands/dotnet-restore/Program.cs

```diff
@@ -35,6 +35,11 @@ public static int Run(string[] args)
                     LocalizableStrings.CmdSourceOptionDescription,
                     CommandOptionType.MultipleValue);
 
+            var runtimeOption = cmd.Option(
+                    $"-r|--runtime <{LocalizableStrings.CmdRuntimeOption}>",
+                    LocalizableStrings.CmdRuntimeOptionDescription,
```

Comment:
In general this looks good to me. But @krwq is right. We have a string freeze right now. Any chance you can re-use one of the strings already available here?